front/lib/swr/actions.ts

```diff
@@ -46,21 +44,17 @@ export function useAgentConfigurationActions(
     }
   );
 
-  // Transform assistant builder actions to agent builder form actions
-  const actions = useMemo((): AgentBuilderAction[] => {
-    if (!data?.actions) {
-      return [];
-    }
-
-    const transformResult = transformAssistantBuilderActionsToFormData(
```

Comment:
If I understand it correctly action card was not shown because somewhere in transformAssistantBuilderActionsToFormData we throw an error, and given that we will use type: "MCP" only, we don't have to transform it anymore??